features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/configuration/SlowFrameListenerConfiguration.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+package com.datadog.android.rum.configuration
+
+/**
+ * Configuration class for the slow frames listener, used to define thresholds for detecting
+ * slow frames, frozen frames, and ANRs (Application Not Responding) in the context of performance monitoring.
+ *
+ * @param maxSlowFramesAmount The maximum number of slow frame records to track. Default is 1000.
+ * @param frozenFrameThresholdNs The threshold (in nanoseconds) used to determine whether a frame is considered frozen.
+ *                               Default is 700 000 000 ns (700ms).
+ * @param continuousSlowFrameThresholdNs The threshold (in nanoseconds) for considering a frame as continuously slow.
+ *                                       Default is 16 666 666 ns (approximately 1/60 fps).
+ * @param anrDuration The duration (in nanoseconds) to consider as an ANR (Application Not Responding).
+ *                    Default is 5 000 000 000 ns (5 seconds).
```

Comment:
should we expose this property or even have ANR reporting as a part of this functionality? it kind of overlaps with existing `trackNonFatalAnrs` and we don't report non-fatal ANRs by default, because there is too much noise.

Also the threshold may vary depending on the component reporting such ANR (e.g. for activity, service and widget it will be different thresholds, also depending on the app state).

Reply:
we need this for Freeze ratio computation. ANR will not reported here, only counted for proper ratio calculation.
ratio are view related and expected to represent visible screens (activity) but not services.

Reply:
renamed from ANR to freezeDuration